install.sh

```diff
@@ -122,7 +123,14 @@ function extract_install () {
     echo "Extracting ${1} (this may take a while)..."
     tar xpf ../"${2}"
     echo "Installing ${1} (this may take a while)..."
-    tar cpf - ./*/* | (cd /; tar xp --keep-directory-symlink -f -)
+    local TarPaths=()
+    for folder in $crew_folders; do
+      if [ -d ."${CREW_PREFIX}"/"$folder" ]; then
+        TarPaths+=(".${CREW_PREFIX}/${folder}")
+      fi
+    done
+    # The following array expansion is intentionally unquoted.
+    tar cpf - ${TarPaths[@]} | (cd /; tar xp --keep-directory-symlink -f -)
```

Comment:
Actually, we can only copy the files under `./usr/local` to fix the problem:
```shell
tar cpf - ./usr/local/* | (cd /; tar xp --keep-directory-symlink -f -)
```

Reply:
Good idea but this would be even better:
```
tar cpf - "${CREW_PREFIX}/*" | (cd /; tar xp --keep-directory-symlink -f -)
```

Reply:
Why glob at all then?  

```tar cpf - "${CREW_PREFIX}" | (cd /; tar xp --keep-directory-symlink -f -)```

Should work just as well, right?

Though again, this does nothing to prevent a package from spewing stuff all over /usr/local

Reply:
Why are you worried about a package spewing files in `/usr/local`?  It should be a dumping grounds with no restrictions.

Reply:
Because _I_ use ```/usr/local``` as a not-dumping ground with various mounts and symlinks to other places. :)

_Also I'm trying to teach a 3 year old to not use places as dumping grounds, and there's probably some part of me trying to tilt at windmills and establish order somewhere._